common-pipeline/datarecord-metadata/datarecord-metadata-v3-schema.json

```diff
@@ -166,9 +169,40 @@
         "runtime_type": {
           "description": "Type of runtime",
           "type": "string"
+        },
+        "fraction_digits": {
+          "description": "Microsecond length for time and timestamp",
+          "type": "number"
+        },
+        "is_key": {
+          "description": "A field is key",
+          "type": "boolean"
+        },
+        "is_signed": {
+          "description": "Signed or Unsigned for number (bigint, integer, smallint, tinyint)",
+          "type": "boolean"
+        },
+        "item_index": {
+          "description": "Item record level",
+          "type": "number"
+        },
+        "min_length": {
```

Comment:
Maybe move this under max_length in the schema?  Also remove the `(binary, bit, char, nchar)` since those are application specific types.

Reply:
Fixed